src/components/LoansByCategory/FeaturedHeroLoan.vue

```diff
@@ -194,7 +194,7 @@ export default {
 			return !!(this.loanUse.length > this.loanUseMaxLength);
 		},
 		showFundraisingThermometer() {
-			return _get(this.experimentData, 'featured_hero_loan_fundraising_thermometer') === 'variant-c';
+			return _get(this.experimentData, 'featured_hero_loan_fundraising_thermometer') === 'variant-b';
```

Comment:
I don't think this matches you code below which has variant-c showing the thermo